client/components/editor/structure/ContentContainers/Container.vue

```diff
@@ -30,17 +30,28 @@
         <content-element
           :set-width="setWidth"
           :dragged="dragged"
-          :element="item" />
+          :element="item"
+          :element-style="getElementStyle(item.contentId)">
+          <div
+            class="active-users-wrapper">
+            <active-users
+              v-if="getActiveUsers('element', item.contentId)"
+              :users="getActiveUsers('element', item.contentId)"
+              :size="26" />
+          </div>
```

Comment:
Seems like we could simplify this by removing wrapping `div` altogether by passing class directly to `ActiveUsers` component. Would that work?